features/utilities/src/main/feature/feature.xml

```diff
@@ -233,7 +237,7 @@
         <bundle>wrap:mvn:org.geotools/gt-opengis/${org.geotools.version}$Bundle-Name=gt-opengis&amp;Bundle-SymbolicName=gt-opengis</bundle>
         <bundle>wrap:mvn:org.geotools/gt-main/${org.geotools.version}$Bundle-Name=gt-main&amp;Bundle-SymbolicName=gt-main</bundle>
         <bundle>wrap:mvn:org.geotools/gt-cql/${org.geotools.version}$Bundle-Name=gt-cql&amp;Bundle-SymbolicName=gt-cql</bundle>
-        <bundle>wrap:mvn:org.geotools/gt-jts-wrapper/${org.geotools.version}$Bundle-Name=gt-jts-wrapper&amp;Bundle-SymbolicName=gt-jts-wrapper</bundle>
+        <bundle>wrap:mvn:org.geotools/gt-jts-wrapper/24.7$Bundle-Name=gt-jts-wrapper&amp;Bundle-SymbolicName=gt-jts-wrapper</bundle>
```

Comment:
Maybe we should make a new version declaration in the pom in case we would like to update the version of this wrapper in the future?